app/controllers/spotlight/catalog_controller.rb

```diff
@@ -201,4 +201,13 @@ def current_page_context
       end
     end
   end
+
+  def additional_export_formats document, format
+    super
+
+    format.solr_json do
+      authorize! :update_solr, @exhibit
```

Comment:
Just out of curiosity, what's the rationale behind requiring this level of permission for viewing this json?